perl got noisy wrt un-escaped { in regexp, shut it up
for texdef: #812154
---
 texmf-dist/scripts/texdef/texdef.pl |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- texlive-extra-2015.20160117.orig/texmf-dist/scripts/texdef/texdef.pl
+++ texlive-extra-2015.20160117/texmf-dist/scripts/texdef/texdef.pl
@@ -388,10 +388,10 @@
             my $protectedmacro = $2;
             unshift @cmds, $protectedmacro;
         }
-        elsif ($macrodef =~ /^\\\@protected\@testopt {?\\.*? }? *(\\\\.*?) /) {
+        elsif ($macrodef =~ /^\\\@protected\@testopt \{?\\.*? \}? *(\\\\.*?) /) {
             unshift @cmds, $1;
         }
-        elsif ($macrodef =~ /^\\\@testopt {?(\\.*?) }?/) {
+        elsif ($macrodef =~ /^\\\@testopt \{?(\\.*?) \}?/) {
             unshift @cmds, $1;
         }
     }
